hydra.yaml

```diff
@@ -0,0 +1,576 @@
+openapi: "3.0.0"
```

Comment:
Maybe this file should be called `hydra-api.yaml` or something? `hydra.yaml` is a bit non-descriptive.